website/docs/vision.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Vision
+sidebar_position: 2
+---
+
+OpenDAL VISION: **access data freely**.
+
+### Free from services
+
+OpenDAL must enable users to access various storage services ranging from `s3` to `dropbox` via it's own native API. It should provide a unified API for accessing all these services.
+
+- Add support for google drive: Good, it allows users to access and manage their data on the google drive.
+- Add support for oss via native API: Good, users can utilize Aliyun's RAM support.
+- Add support for [supabase storage](https://supabase.com/docs/guides/storage): Good, users can visit supabase storage now!
+
+
+- Add support for gcs via XML API: Bad, gcs has native JSON API which more powerful
+- Add support for MySQL/PostgreSQL: Bad, relational DBMS provides data types such as BLOB, but they are often not used as a storage service.
```

Comment:
Need to handle case correctly. (plus maybe follow with the full name?)

Reply:
Can you point out more details? I didn't get this review comment.

Reply:
Like `Google Drive`? Or `GCS(Google Cloud Storage)`?

Reply:
Fixed.